source/index.md

```diff
@@ -849,6 +975,17 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add payment methods
+
+Payment methods have been added to the authorizations resource. Also, a few
+parameters and an endpoint have been deprecated. Refer to [the documentation
+source code changes](https://github.com/clearhaus/gateway-api-docs/pull/53/files)
+for the exact documentation change.
+
+Please notice that there is no major version number change, so we stay backwards
+compatible until the deprecations take effect. Expect the deprecations to happen
+within a month or so; it will be announced separately.
```

Comment:
Perhaps `s/a month/two months/` since we aim for mid-October?

@kse-clearhaus WDYT?

Reply:
:+1: